west.yml

```diff
@@ -144,7 +144,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 5936aa15ddceaeed703c9adebb831c6d740fa8d8
+      revision: pull/1736/head
```

Comment:
could you update dragoon sha as well: https://github.com/nrfconnect/sdk-nrf/pull/22105/files#diff-d7f4082837dc3cb270fcf1abd68188216fdadee294e01611b8ca59ddea7fdbb3R185

Reply:
I will create separate PR then